src/platforms/hosted/dap_swd.c

```diff
@@ -119,59 +128,94 @@ static void dap_swd_seq_out(const uint32_t tms_states, const size_t clock_cycles
 		DEBUG_ERROR("dap_swd_seq_out failed\n");
 }
 
+/*Fallback when SWD sequence to implemented*/
+static void dap_swj_seq_out(const uint32_t tms_states, const size_t clock_cycles)
+{
+	const uint8_t data[4] = {(tms_states >> 0) & 0xff, (tms_states >> 8) & 0xff,
```

Comment:
Don't do this manually, use the write_le4() function from buffer_utils.h (already included above).

Reply:
Will do.